patroni/postgresql/__init__.py

```diff
@@ -508,7 +508,7 @@ def _do_stop(self, mode, block_callbacks, checkpoint, on_safepoint):
             self.set_state('stopping')
 
         # Send signal to postmaster to stop
-        success = postmaster.signal_stop(mode)
+        success = postmaster.signal_stop(mode, self._bin_dir)
```

Comment:
I think it would be better to pass `self.pgcommand('pg_ctl')` instead of _bin_dir.